docs/reference/components/sinks/kafka.cue

```diff
@@ -16,6 +16,13 @@ components: sinks: kafka: {
 		buffer: enabled:      true
 		healthcheck: enabled: true
 		send: {
+			batch: {
```

Comment:
@binarylogic When I run this through `check-docs` locally I see:

```cue
components.sinks.kafka.features.send: field `batch` not allowed:
    ./docs/reference.cue:133:12
    ./docs/reference.cue:311:13
    ./docs/reference.cue:311:29
    ./docs/reference.cue:380:16
    ./docs/reference.cue:866:19
    ./docs/reference.cue:869:14
    ./docs/reference/components.cue:4:2
    ./docs/reference/components/sinks/kafka.cue:19:4
```

Which I think is due to https://github.com/timberio/vector/blob/7ff9cbf7bc9d77d3b9f019fee8897065a9434be5/docs/reference.cue#L387-L396

Could you help me understand how to communicate this to cue?

Kafka is optionally a batching stream sink if the user configures it, and has no required `timeout_secs` (defaulting to what rdkafka chooses).

Reply:
So if a user does not configure `batch` settings then it is effectively a streaming sink? No batching at all?

Reply:
Correct, this does not change the behavior of the sink at all, this just aliases batch options to already configurable kafka options.

Reply:
@binarylogic Friendly re-ping on this. :)